models/black_lodge/src/training/train_model.py

```diff
@@ -0,0 +1,69 @@
+import sys
+from pathlib import Path
+import pandas as pd
+
+PATH = Path(__file__) 
+sys.path.insert(0, str(Path(*[i for i in PATH.parts[:PATH.parts.index("views_pipeline")+1]]) / "common_utils"))
+from set_path import setup_project_paths, setup_data_paths, setup_artifacts_paths
+setup_project_paths(PATH)
+
+from xgboost import XGBRFRegressor
+
+from views_partitioning.data_partitioner import DataPartitioner
+from views_forecasts.extensions import *
+from stepshift.views import StepshiftedModels
+from views_stepshift.run import ViewsRun
+from hurdle_model import HurdleRegression
+from get_data import get_partition_data
+
+from config_meta import get_meta_config
+from config_hyperparameters import get_hp_config
+
+
+def train(model_config, para_config):
```

Comment:
You can refer to my new training function (I haven't merged the main so it's branch more_model and you can check orange_pasta). In short, we don't train three models together but train a specific one based on the arguments instead.